compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -291,20 +291,6 @@ class JSExportTest extends DirectTest with TestHelpers {
   }
 
   @Test
-  def infoExportLocal: Unit = {
-
-    """
-    class A(@JSExport val x: Int)
-    """ hasErrors
-    """
-      |newSource1.scala:3: error: You may not export a local definition. To export a (case) class field, use the meta-annotation scala.annotation.meta.field like this: @(JSExport @field).
-      |    class A(@JSExport val x: Int)
-      |             ^
```

Comment:
There is no error anymore. It now compiles. There are new positive tests for this in `ExportsTest`.